src/pages/SubmissionDetails.js

```diff
@@ -80,7 +81,12 @@ export const SubmissionDetails = ({ submissionId }) => {
   const specifyRunSet = _.filter(r => r.run_set_id === submissionId, runSetData)
   const methodId = specifyRunSet[0]?.method_id
   const getSpecificMethod = _.filter(m => m.method_id === methodId, methodsData)
-  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, runsData)
+
+  // hardcoded to return error_messages since this is the only option we currently want to filter by
+  const errorStates = ['SYSTEM_ERROR', 'EXECUTOR_ERROR']
+  const filteredPreviousRuns = filterOption ? _.filter(r => errorStates.includes(r.state), runsData) : runsData
+  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, filteredPreviousRuns)
+  const filterOptions = ['Error']
```

Comment:
We probably want a "None" option so we can turn off the filter again without reloading the page?